rama-http/Cargo.toml

```diff
@@ -58,6 +58,7 @@ tokio = { workspace = true, features = ["macros", "fs", "io-std"] }
 tokio-util = { workspace = true, features = ["io"] }
 tracing = { workspace = true }
 uuid = { workspace = true, features = ["v4"] }
+trie-rs = "0.4.2"
```

Comment:
add it first as a dependency under `./Cargo.toml` please (workspace dependency, afterwards you can add it here as `trie-rs = { workspace = true }`.

The workspace dependency would be defined as:

```
trie-rs = { version = "0.4", features = ["serde"] }
```